cypress/integration/osd/check_default_page.js

```diff
@@ -39,6 +39,5 @@ describe('verify default landing page work for bwc', () => {
   
   it('the overview page is set as the default landing pag', () => {
```

Comment:
can we fix this to say `default landing page`

Reply:
will fix